### PR TITLE
I added haiku to the project

### PR DESCRIPTION
**Pull request by CJestrip-89**

### CODE REVIEW

File: _haikus/haikuchris.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: haiku
+title: my firs time using git
+author: chris
+---
+
+today's History
+tommorow is progress
```
Comment (beardicus) [inline _haikus/haikuchris.md]: this line should be seven syllables.

File: _haikus/haikuchris.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+all about consistency
```
Comment (beardicus) [inline _haikus/haikuchris.md]: this line should be five syllables.

File: _haikus/haikuchris.md (hunk 1 above)
Comment (beardicus) [inline _haikus/haikuchris.md]: this line should be seven syllables

File: _haikus/haikuchris.md (hunk 2 above)
Comment (beardicus) [inline _haikus/haikuchris.md]: this line should be five syllables.

File: _haikus/haikuchris.md (hunk 1 above)
Comment (CJestrip-89) [reply _haikus/haikuchris.md]: thank you

Comment (CJestrip-89) [reply _haikus/haikuchris.md]: How do i upload my updates i'm having trouble, because this my first time using git and github

Comment (ltagliaferri) [reply _haikus/haikuchris.md]: You should be able to modify your file locally and just update this pull request with a new commit. If it is easier, you can close this pull request and open a new one.

**Pull request closed without merging**